templates/registration/login.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Login</h1>
+  
+  {% if form.errors %}
+    <p style="...">Invalid credentials</p>
+  {% endif %}
+
+  <form method="post" action="{% url 'login' %}">
+    {% csrf_token %}
+    {{ form.as_p }}
+    <input type="submit" value="Submit">
+  </form>
```

Comment:
According to the checklist, you should use 2 whitespaces for indentation in your `.html` files. It seems like you're using more than 2 spaces here. Remember, consistent indentation improves code readability.